### PR TITLE
Cli: Configure Docker Files

### PR DESCRIPTION
**Pull request by yekalkan**

### CODE REVIEW

File: framework/src/Volo.Abp.Cli.Core/Volo/Abp/Cli/ProjectBuilding/Building/Steps/TemplateCodeDeleteStep.cs [hunk 1]
```diff
@@ -13,6 +13,8 @@ public override void Execute(ProjectBuildContext context)
                 file.Name.EndsWith(".cshtml") ||
                 file.Name.EndsWith(".json") ||
                 file.Name.EndsWith(".gitignore") ||
+                file.Name.EndsWith(".yml") ||
+                file.Name.EndsWith(".ps1") ||
```
Comment (gterdem) [inline framework/src/Volo.Abp.Cli.Core/Volo/Abp/Cli/ProjectBuilding/Building/Steps/TemplateCodeDeleteStep.cs:17]: We need to add .json also:
https://github.com/volosoft/volo/blob/fe5640e781607bd2419816b307c3f2e0ab8a25a8/abp/templates/app-pro/aspnet-core/etc/docker/dynamic-env.json

If this this json file is not valid, remote environment feature is not working for angular.

File: framework/src/Volo.Abp.Cli.Core/Volo/Abp/Cli/ProjectBuilding/Templates/App/AppNoLayersTemplateBase.cs [hunk 2]
```diff
@@ -102,4 +95,35 @@ public override IEnumerable<ProjectBuildPipelineStep> GetCustomSteps(ProjectBuil
 
         return steps;
     }
+
+    private void ConfigureDockerFiles(ProjectBuildContext context, List<ProjectBuildPipelineStep> steps)
+    {
```
Comment (gterdem) [inline framework/src/Volo.Abp.Cli.Core/Volo/Abp/Cli/ProjectBuilding/Templates/App/AppNoLayersTemplateBase.cs:100]: We can remove [dynamic-env.json](https://github.com/volosoft/volo/blob/fe5640e781607bd2419816b307c3f2e0ab8a25a8/abp/templates/app-pro/aspnet-core/etc/docker/dynamic-env.json) if the solution is **not angular**.

**Review by gterdem — APPROVED** [review]

File: framework/src/Volo.Abp.Cli.Core/Volo/Abp/Cli/ProjectBuilding/Building/Steps/TemplateCodeDeleteStep.cs (hunk 1 above)
Comment (yekalkan) [reply framework/src/Volo.Abp.Cli.Core/Volo/Abp/Cli/ProjectBuilding/Building/Steps/TemplateCodeDeleteStep.cs:17]: It is already added. (line 14)

File: framework/src/Volo.Abp.Cli.Core/Volo/Abp/Cli/ProjectBuilding/Templates/App/AppNoLayersTemplateBase.cs (hunk 2 above)
Comment (yekalkan) [reply framework/src/Volo.Abp.Cli.Core/Volo/Abp/Cli/ProjectBuilding/Templates/App/AppNoLayersTemplateBase.cs:100]: Already done.

**Pull request merged**